operator/v1/types.go

```diff
@@ -45,6 +45,21 @@ type ClusterManagerSpec struct {
 	// NodePlacement enables explicit control over the scheduling of the deployed pods.
 	// +optional
 	NodePlacement NodePlacement `json:"nodePlacement,omitempty"`
+
+	// HubDeployMode represents the mode used to deploy hub components. There are two mode: mode `Unite` as default, and mode `Separate`.
+	// In Separate mode, a "shell-hub" holds all CRDs, and "worker-hub" has all deployment running in it.
+	// +kubebuilder:default=Unite
+	HubDeployMode string `json:"hubDeployMode"`
```

Comment:
What is shell-hub? I would name it as `DeployMode`

Reply:
It represents the cluster installed with CRD. I named them "shell" and "worker" currently but there could be other suitable names.

Reply:
For the DeployMode, are we going to use this one for both cluster-manager deploy and klusterlet deploy?